pr_testing/run-pr-unittests.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash -ex
+source $(dirname $0)/setup-pr-test-env.sh
+
+echo 'UNIT_TEST_RESULTS;NOTRUN' > ${RESULTS_DIR}/unittest.txt
```

Comment:
rename `unittest.txt` to `unittestGPU.txt` every where otherwise it will override the default unittest.txt generated by CPU unit tests